hydra-node/test/Hydra/LoggingSpec.hs

```diff
@@ -0,0 +1,16 @@
+module Hydra.LoggingSpec where
+
+import Data.Aeson (object, (.=))
+import Hydra.Logging (Verbosity (Verbose), traceWith, withTracer)
+import Hydra.Prelude
+import System.IO.Silently (capture_)
+import Test.Hspec (Spec, describe, it, shouldContain)
+
+spec :: Spec
+spec = describe "Logging Tracer" $ do
+  it "dumps logs to stdout in JSON with timestamp" $ do
+    captured <- capture_ $ do
```

Comment:
@abailly-iohk This is what I should've used when capturing and printing the PAB output. Maybe then it would work better.